tests/model_fields/tests.py

```diff
@@ -698,16 +706,30 @@ class BigIntegerFieldTests(IntegerFieldTests):
     documented_range = (-9223372036854775808, 9223372036854775807)
 
 
+@unittest.skipIf(connection.vendor == 'mysql', "Test not for MySQL")
```

Comment:
This test checks the "documented ranges" that all backends accept according to ref/models/fields, however, it will be a problem for any third-party backends that don't support that range, so maybe using the value from `connections.ops.integer_field_ranges` is a better way to go. Then we won't need the MySQL specific version.